client-java/controller/src/main/java/org/evomaster/client/java/controller/internal/EMController.java

```diff
@@ -158,7 +158,7 @@ public Response getSutInfo(@Context HttpServletRequest httpServletRequest) {
 
         assert trackRequestSource(httpServletRequest);
 
-        if (!noKillSwitch(() -> sutController.verifySqlConnection())) {
+        if (!noKillSwitch(sutController::verifySqlConnection)) {
```

Comment:
personally, I do not like the use of `::`. if someone wants to use it, I don't mind, but I don't think we should refactor existing occurrences of it